src/framework/dropdown/DropdownGroup.jsx

```diff
@@ -0,0 +1,101 @@
+
+import React, {
+  Component,
+  PropTypes,
+} from 'react';
+import classNames from 'classnames';
+
+import BaseDropdown from '../base/dropdown/BaseDropdown.jsx';
+import DropdownDot from './dropdownDot/DropdownDot.jsx';
+import DropdownOption from './DropdownOption.jsx';
+
+export {
+  default as DropdownDot,
+} from './dropdownDot/DropdownDot.jsx';
+
+export {
+  default as DropdownOption,
+} from './DropdownOption.jsx';
```

Comment:
Exports already happening in `Button`, no need for exporting (even thinking if this should somehow fail).